### PR TITLE
fix(axiosconfig): non-existant jwt should not trigger a token refresh

### PR DESCRIPTION
**Pull request by douglashall**

### CODE REVIEW

File: src/AuthenticatedAPIClient/axiosConfig.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ function applyAxiosInterceptors(authenticatedAPIClient) {
     const isAuthUrl = authenticatedAPIClient.isAuthUrl(originalRequest.url);
     const accessToken = authenticatedAPIClient.getDecodedAccessToken();
     const tokenExpired = authenticatedAPIClient.isAccessTokenExpired(accessToken);
-    if (isAuthUrl || !tokenExpired) {
+    if (isAuthUrl || !accessToken || !tokenExpired) {
```
Comment (nasthagiri) [inline src/AuthenticatedAPIClient/axiosConfig.js]: How did this manifest itself?  Will there ever be a case where this code is called and no JWT is present?
Does `isAccessTokenExpired` handle NULL accessToken?

Comment (douglashall) [reply src/AuthenticatedAPIClient/axiosConfig.js]: This manifested itself in the tests of consuming apps. The apps' tests mock the API client and do not scaffold the request with a JWT.

It is possible that this code would be called when there is not JWT present. In practice this should not happen if the app is checking authentication status before rendering, but the app is in control of this.

`isAccessTokenExpired` considers a null accessToken to be expired.

File: src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx [hunk 2]
```diff
@@ -268,12 +270,16 @@ describe('AuthenticatedAPIClient request headers', () => {
 describe('AuthenticatedAPIClient ensureValidJWTCookie request interceptor', () => {
   [
     /* isAuthUrl, isAccessTokenExpired, queueRequests, rejectRefreshAccessToken, expects */
```
Comment (robrap) [inline src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx]: Update comment?

File: src/AuthenticatedAPIClient/axiosConfig.js (hunk 1 above)
Comment (nasthagiri) [reply src/AuthenticatedAPIClient/axiosConfig.js]: Got it. Thanks for the clarification.

**Pull request merged**